ng2ts/tests/codelabTest.ts

```diff
@@ -56,6 +56,11 @@ describe('Component', () => {
      * e.g. if the user created teh class, but haven't exported it this
      * test will still pass.
      */
+
+    const codeString = typescript_intro_Codelab_ts_AST.text;
+    const regEx = RegExp(/(Codelab) *\(\)/g);
+    const noParentheses = codeString.search(regEx) === -1;
+    chai.expect(noParentheses).equals(true);
```

Comment:
Not familiar with chai, but can't you do `expect(codeString).not.toMatch(/(Codelab) *\(\)/g)` like jasmine?